### PR TITLE
lib/ivykis: update to 0.42

### PR DESCRIPTION
**Pull request by bazsi**
To keep up with upstream.

Signed-off-by: Balazs Scheidler <balazs.scheidler@balabit.com>

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build FAILURE, the tests were executed on test branch: master and test suite: functions

**Comment by presidento** [comment]
@kira-syslogng retest this please

**Comment by kira-syslogng** [comment]
Build FAILURE, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by bazsi** [comment]
As it seems @kira-syslogng agrees that the new ivykis version works all right, so we should progress with time and merge this beofre we get too close to release.

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Comment by lbudai** [comment]
here are the installers for the perftest: https://build.opensuse.org/package/show/home:laszlo_budai:syslog-ng-upstream/syslog-ng-3.10#

File: configure.ac [hunk 1]
```diff
@@ -44,7 +44,7 @@ GLIB_MIN_VERSION="2.10.1"
 OPENSSL_MIN_VERSION="0.9.8"
 LIBDBI_MIN_VERSION="0.9.0"
 LIBRABBITMQ_MIN_VERSION="0.5.3"
-IVYKIS_MIN_VERSION="0.36.1"
```
Comment (MrAnno) [inline configure.ac]: Is this a good idea? The "system" ivykis won't work on most platforms.

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Pull request merged**